README
NAME
    Task::BeLike::GENEHACK - individuality via conformity

VERSION
    version 0.2

TASK CONTENTS
  Modules I use
   App::Ack
   App::BashComplete
   App::Cmd
   App::MiseEnPlace
   App::Nopaste
   App::cpanminus
   App::cpanoutdated
   App::perlbrew
   Audio::M4P::QuickTime
   Bash::Completion
   Bash::Completion::Plugins::cpanm
   Bash::Completion::Plugins::perlbrew
   CPAN::Mini
   DBD::SQLite
   Data::Dump::Streamer
   DateTime
   Devel::Cover
   Devel::REPL
   Dist::Zilla::App::Command::cover
   Dist::Zilla::PluginBundle::GENEHACK
   File::Slurp
   Imager
   List::MoreUtils
   Modern::Perl
   Module::CoreList
   Module::Install
   Module::Which
   Moose
   MooseX::App::Cmd
   MooseX::Declare
   MooseX::amine
   Perl::Tidy
   Pod::Coverage::TrustPod
   Task::Catalyst
   Task::Kensho
   Task::Kensho::CLI
   Task::Kensho::Config
   Task::Kensho::Dates
   Task::Kensho::Hackery
   Task::Kensho::ModuleDev
   Task::Kensho::WebDev
   Task::Kensho::XML
   Task::KiokuDB
   Test::Class
   Test::Exception
   Test::MockObject::Extends
   Test::Most
   Test::Pod
   Test::Pod::Coverage
   Text::FindIndent
   Try::Tiny
   XML::LibXML
   XML::Simple
   XML::Writer
   YAML
   autodie
   local::lib
   namespace::autoclean
AUTHOR
    John SJ Anderson <genehack@genehack.org>

COPYRIGHT AND LICENSE
    This software is copyright (c) 2011 by John SJ Anderson.

    This is free software; you can redistribute it and/or modify it under
    the same terms as the Perl 5 programming language system itself.



Changes
Revision history for Task-BeLike-GENEHACK

0.2       2011-07-19 07:34:54 America/New_York

  Add modules:

    * Moose
    * Pod::Coverage::TrustPod
    * Audio::M4P::QuickTime
    * Imager
    * Bash::Completion (and plugins)
    * Text::FindIndent

0.1.1     2011-04-22 21:26:23 America/New_York

  Had a stupid typo in namespace::autoclean...

0.1       2011-04-22 20:49:17 America/New_York

  Initial release -- test builds of 5.14.0 finally got me to put this
  together rather than installing everything one-off...



lib/Task/BeLike/GENEHACK.pm
package Task::BeLike::GENEHACK;
BEGIN {
  $Task::BeLike::GENEHACK::VERSION = '0.2';
}
BEGIN {
  $Task::BeLike::GENEHACK::AUTHORITY = 'cpan:GENEHACK';
}
# ABSTRACT: individuality via conformity

use strict;
use warnings;
use 5.010;


1;

__END__
=pod

=head1 NAME

Task::BeLike::GENEHACK - individuality via conformity

=head1 VERSION

version 0.2

=head1 TASK CONTENTS

=head2 Modules I use

=head3 L<App::Ack>

=head3 L<App::BashComplete>

=head3 L<App::Cmd>

=head3 L<App::MiseEnPlace>

=head3 L<App::Nopaste>

=head3 L<App::cpanminus>

=head3 L<App::cpanoutdated>

=head3 L<App::perlbrew>

=head3 L<Audio::M4P::QuickTime>

=head3 L<Bash::Completion>

=head3 L<Bash::Completion::Plugins::cpanm>

=head3 L<Bash::Completion::Plugins::perlbrew>

=head3 L<CPAN::Mini>

=head3 L<DBD::SQLite>

=head3 L<Data::Dump::Streamer>

=head3 L<DateTime>

=head3 L<Devel::Cover>

=head3 L<Devel::REPL>

=head3 L<Dist::Zilla::App::Command::cover>

=head3 L<Dist::Zilla::PluginBundle::GENEHACK>

=head3 L<File::Slurp>

=head3 L<Imager>

=head3 L<List::MoreUtils>

=head3 L<Modern::Perl>

=head3 L<Module::CoreList>

=head3 L<Module::Install>

=head3 L<Module::Which>

=head3 L<Moose>

=head3 L<MooseX::App::Cmd>

=head3 L<MooseX::Declare>

=head3 L<MooseX::amine>

=head3 L<Perl::Tidy>

=head3 L<Pod::Coverage::TrustPod>

=head3 L<Task::Catalyst>

=head3 L<Task::Kensho>

=head3 L<Task::Kensho::CLI>

=head3 L<Task::Kensho::Config>

=head3 L<Task::Kensho::Dates>

=head3 L<Task::Kensho::Hackery>

=head3 L<Task::Kensho::ModuleDev>

=head3 L<Task::Kensho::WebDev>

=head3 L<Task::Kensho::XML>

=head3 L<Task::KiokuDB>

=head3 L<Test::Class>

=head3 L<Test::Exception>

=head3 L<Test::MockObject::Extends>

=head3 L<Test::Most>

=head3 L<Test::Pod>

=head3 L<Test::Pod::Coverage>

=head3 L<Text::FindIndent>

=head3 L<Try::Tiny>

=head3 L<XML::LibXML>

=head3 L<XML::Simple>

=head3 L<XML::Writer>

=head3 L<YAML>

=head3 L<autodie>

=head3 L<local::lib>

=head3 L<namespace::autoclean>

=head1 AUTHOR

John SJ Anderson <genehack@genehack.org>

=head1 COPYRIGHT AND LICENSE

This software is copyright (c) 2011 by John SJ Anderson.

This is free software; you can redistribute it and/or modify it under
the same terms as the Perl 5 programming language system itself.

=cut



Makefile.PL

use strict;
use warnings;

use 5.010;

use ExtUtils::MakeMaker 6.30;



my %WriteMakefileArgs = (
  'ABSTRACT' => 'individuality via conformity',
  'AUTHOR' => 'John SJ Anderson <genehack@genehack.org>',
  'BUILD_REQUIRES' => {
    'File::Find' => '0',
    'File::Temp' => '0',
    'Test::More' => '0'
  },
  'CONFIGURE_REQUIRES' => {
    'ExtUtils::MakeMaker' => '6.30'
  },
  'DISTNAME' => 'Task-BeLike-GENEHACK',
  'EXE_FILES' => [],
  'LICENSE' => 'perl',
  'NAME' => 'Task::BeLike::GENEHACK',
  'PREREQ_PM' => {
    'App::Ack' => '0',
    'App::BashComplete' => '0',
    'App::Cmd' => '0',
    'App::MiseEnPlace' => '0',
    'App::Nopaste' => '0',
    'App::cpanminus' => '0',
    'App::cpanoutdated' => '0',
    'App::perlbrew' => '0',
    'Audio::M4P::QuickTime' => '0',
    'Bash::Completion' => '0',
    'Bash::Completion::Plugins::cpanm' => '0',
    'Bash::Completion::Plugins::perlbrew' => '0',
    'CPAN::Mini' => '0',
    'DBD::SQLite' => '0',
    'Data::Dump::Streamer' => '0',
    'DateTime' => '0',
    'Devel::Cover' => '0',
    'Devel::REPL' => '0',
    'Dist::Zilla::App::Command::cover' => '0',
    'Dist::Zilla::PluginBundle::GENEHACK' => '0',
    'File::Slurp' => '0',
    'Imager' => '0',
    'List::MoreUtils' => '0',
    'Modern::Perl' => '0',
    'Module::CoreList' => '0',
    'Module::Install' => '0',
    'Module::Which' => '0',
    'Moose' => '0',
    'MooseX::App::Cmd' => '0',
    'MooseX::Declare' => '0',
    'MooseX::amine' => '0',
    'Perl::Tidy' => '0',
    'Pod::Coverage::TrustPod' => '0',
    'Task::Catalyst' => '0',
    'Task::Kensho' => '0',
    'Task::Kensho::CLI' => '0',
    'Task::Kensho::Config' => '0',
    'Task::Kensho::Dates' => '0',
    'Task::Kensho::Hackery' => '0',
    'Task::Kensho::ModuleDev' => '0',
    'Task::Kensho::WebDev' => '0',
    'Task::Kensho::XML' => '0',
    'Task::KiokuDB' => '0',
    'Test::Class' => '0',
    'Test::Exception' => '0',
    'Test::MockObject::Extends' => '0',
    'Test::Most' => '0',
    'Test::Pod' => '0',
    'Test::Pod::Coverage' => '0',
    'Text::FindIndent' => '0',
    'Try::Tiny' => '0',
    'XML::LibXML' => '0',
    'XML::Simple' => '0',
    'XML::Writer' => '0',
    'YAML' => '0',
    'autodie' => '0',
    'local::lib' => '0',
    'namespace::autoclean' => '0'
  },
  'VERSION' => '0.2',
  'test' => {
    'TESTS' => 't/*.t'
  }
);


unless ( eval { ExtUtils::MakeMaker->VERSION(6.56) } ) {
  my $br = delete $WriteMakefileArgs{BUILD_REQUIRES};
  my $pp = $WriteMakefileArgs{PREREQ_PM};
  for my $mod ( keys %$br ) {
    if ( exists $pp->{$mod} ) {
      $pp->{$mod} = $br->{$mod} if $br->{$mod} > $pp->{$mod};
    }
    else {
      $pp->{$mod} = $br->{$mod};
    }
  }
}

delete $WriteMakefileArgs{CONFIGURE_REQUIRES}
  unless eval { ExtUtils::MakeMaker->VERSION(6.52) };

WriteMakefile(%WriteMakefileArgs);



